<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>371</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="dataLabelHazard">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string>&lt;empty&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="pictureLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/rc/busy.bmp);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>400</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix1/rc/logo_bottom.gif);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="itemLabelCode">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Barcode:</string>
    </property>
   </widget>
   <widget class="QLabel" name="itemLabelProduct">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Product:</string>
    </property>
   </widget>
   <widget class="QLabel" name="itemLabelManufacturer">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Manufacturer:</string>
    </property>
   </widget>
   <widget class="QLabel" name="itemLabelSponsor">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>430</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Main software sponsor:</string>
    </property>
   </widget>
   <widget class="QLabel" name="dataLabelProduct">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>260</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;empty&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="dataLabelManufacturer">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;empty&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="dataLabelBarcode">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;empty&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
